Phase1/src/test/java/TestClass.java

```diff
@@ -0,0 +1,154 @@
+        ByteArrayInputStream in2 = new ByteArrayInputStream("cat\nnothing\nexit".getBytes());
+        System.setIn(in2);
+
+        SearchEngine searchEngine = new SearchEngine();
+        searchEngine.run();
+    }
+
+
+    @Test
+    public void stemmerTest (){
+        FileReader fileReader = new FileReader();
+        InvertedIndex invertedIndex = new InvertedIndex().tokenizeFiles(fileReader.readingFiles("src/test/DocsForTest"));
+
+        UserInput userInput = new UserInput("played actional national enci anci izer bli alli entli eli " +
+                "ousli ization ation ator alism iveness fulness ousness aliti iviti biliti logi icate ative alize " +
+                "iciti ical ful ness er ic iti ive ize");
+        TreeSet<String> files = invertedIndex.query(userInput);
+        Assertions.assertTrue(files.isEmpty());
+    }
+
+
+    @Test
+    public void stemmerTest2 (){
+        FileReader fileReader = new FileReader();
+        InvertedIndex invertedIndex = new InvertedIndex().tokenizeFiles(fileReader.readingFiles("src/test/DocsForTest"));
+
+        UserInput userInput = new UserInput("hello eed piuic");
+        TreeSet<String> files = invertedIndex.query(userInput);
+        Assertions.assertTrue(files.isEmpty());
+    }
+
+
+    @Test
+    public void mainTest (){
+        ByteArrayInputStream in2 = new ByteArrayInputStream("cat\nnothing\nexit".getBytes());
+        System.setIn(in2);
+
+        String[] args = new String[1];
+        args[0] = "who";
+        Main.main(args);
+    }
+
+
+    @Test
+    public void positiveTest (){
+        FileReader fileReader = new FileReader();
+        InvertedIndex invertedIndex = new InvertedIndex().tokenizeFiles(fileReader.readingFiles("src/test/DocsForTest"));
+
+        UserInput userInput = new UserInput("+elephant");
+        TreeSet<String> files = invertedIndex.query(userInput);
+        Assertions.assertTrue(files.isEmpty());
+    }
+
+
+    @Test
+    public void minusTest (){
+        FileReader fileReader = new FileReader();
+        InvertedIndex invertedIndex = new InvertedIndex().tokenizeFiles(fileReader.readingFiles("src/test/DocsForTest"));
+
+        UserInput userInput = new UserInput("-elephant");
+        TreeSet<String> files = invertedIndex.query(userInput);
+        Assertions.assertTrue(files.isEmpty());
+    }
+
+
+    @Test
+    public void exceptionTest (){
+        FileReader fileReader = new FileReader();
+        try {
+            fileReader.readingFiles("noWhere");
+        }catch (Exception e){
+            e.printStackTrace();
+        }
+    }
+
+
+    @Test
+    public void emptyTest (){
+        FileReader fileReader = new FileReader();
+        InvertedIndex invertedIndex = new InvertedIndex().tokenizeFiles(fileReader.readingFiles("src/test/DocsForTest"));
+
+        UserInput userInput = new UserInput(" ");
+        TreeSet<String> files = invertedIndex.query(userInput);
+        Assertions.assertTrue(files.isEmpty());
+    }
+
+
+
+}
```

Comment:
This class is redundant actually. You should move methods to their relevant classes.
For example, the first method( gettingFileAndReadIt ()) should be moved to FileReader class.